rj_common/include/rj_common/utils.hpp

```diff
@@ -24,18 +23,15 @@ inline void debug_log_if(const std::string& e, bool condition) {
 }
 
 template <class T,
-          typename std::enable_if<std::is_base_of<std::exception, T>::value,
-                                  int>::type = 0>
+          typename std::enable_if<std::is_base_of<std::exception, T>::value, int>::type = 0>
 inline void debug_throw(const T& e) {
     debug_log(e);
     if (kThrowDebugExceptions) {
         throw e;
     }
 }
 
-inline void debug_throw(const std::string& string) {
-    debug_throw(std::runtime_error(string));
-}
+inline void debug_throw(const std::string& string) { debug_throw(std::runtime_error(string)); }
```

Comment:
Nit: Shouldn't these be in `rj_utils/logging`?

Reply:
Done.